src/basic/parse-util.c

```diff
@@ -758,3 +759,44 @@ int parse_oom_score_adjust(const char *s, int *ret) {
         *ret = v;
         return 0;
 }
+
+int parse_crash_chvt(const char *value, int *data) {
+        int b;
+
+        if (safe_atoi(value, data) >= 0)
+                return 0;
+
+        b = parse_boolean(value);
+        if (b < 0)
+                return b;
+
+        if (b > 0)
+                *data = 0; /* switch to where kmsg goes */
+        else
+                *data = -1; /* turn off switching */
+
+        return 0;
+}
```

Comment:
I'd prefer to try try keep `basic/` minimal. We didn't always do that in the past, but I think we should. Those two functions are only used in core/, and only makes sense there, so I'd prefer to stash them in on the files there or even add a new file for parsing helpers.

Reply:
Makes sense. Since both functions are somehow related to config parsing, would it suffice to stash them in `load-fragment.[ch]` as well or you prefer a separate file?